packages/client/src/object/convertWireToOsdkObjects.test.ts

```diff
@@ -173,7 +179,37 @@ describe("convertWireToOsdkObjects", () => {
     expect(prototypeBefore).not.toBe(prototypeAfter);
   });
 
-  it("updates interface when underlying changes", async () => {
+  it("works even with unknown apiNames - new", async () => {
```

Comment:
Basically, all the tests are the same, I just added duplicates to test the new functionality, while retaining the old stuff since it's still technically supported. Once we rip out the old apis, we can clean up these tests too